### PR TITLE
#4377 Substitute com.jfrog.bintray for com.novoda:bintray-release

### PR DESCRIPTION
**Pull request by chia7712**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4377)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by chia7712** [comment]
@vitojeng this PR includes the function of installing jar to local repo. Could you take a look?

File: build.gradle [hunk 1]
```diff
@@ -172,53 +166,73 @@ subprojects {
...
-    }
-
-    // this is a hacky workaround that we have to override the internal variables in order to deploy the sources/tests jar ... by chia
-    // see https://github.com/novoda/bintray-release/blob/master/plugin/core/src/main/groovy/com/novoda/gradle/release/MavenPublicationAttachments.groovy
-    // for the internal variables
-    tasks.whenTaskAdded { Task task ->
-      if (name == "generateSourcesJarForMavenPublication") {
-        from sourceSets.main.allSource
+    apply plugin: "com.jfrog.bintray"
+    apply plugin: 'maven'
```
Comment (vitojeng) [inline build.gradle]: Does this line still need?
I remove this line and work fine with `publishToMavenLocal` task.

**Comment by vitojeng** [comment]
> @vitojeng this PR includes the function of installing jar to local repo. Could you take a look?

@chia7712 Test in my machine with following task:
```
$ gradle clean build publishToMavenLocal -x test -PskipManager
```

It seems work well.

**Comment by vitojeng** [comment]
Thanks @chia7712 for the great PR!

File: build.gradle (hunk 1 above)
Comment (chia7712) [reply build.gradle]: not sure. the dependency is recommended by official document :)

**Pull request merged**